### PR TITLE
Bugfix/some specs fixed

### PR DESCRIPTION
**Pull request by sgomezg**
#### Description
* SPARKTA-100 CubeMakeSpec bug solved (#748). 
* Code formatted. 
* Dependencies sorted and revised to fix clean build issues. 
* StreamingContextIT build solved.
* SPARKTA-105 DateOperationsSpec fixed

#### Reviewer
@arinconstrio @danielcsant @anistal 

#### Test
Removed unused test, all green including AT

#### Coverage
Statement Coverage	57.52% 	 
Branch Coverage	43.34%

#### Scalastyle
No info

### CODE REVIEW

File: sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala [hunk 1]
```diff
@@ -44,13 +44,12 @@ class ValidatingPropertyMap[K, V](val m: Map[K, V]) {
         }))
   }
 
-  def getConnectionChain(key: K): Seq[Map[String, String]] = {
-    m.get(key) match {
-      case Some(value) => JSON.parseFull(value.asInstanceOf[JsoneyString].string)
-        .get.asInstanceOf[Seq[Map[String, String]]]
+  def getConnectionChain(key: K): Seq[Map[String, String]] =
```
Comment (arinconstrio) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala]: I think that this refactor is out of scope

Comment (sgomezg) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala]: Change reverted

File: aggregator/src/test/scala/com/stratio/sparkta/aggregator/CubeMakerSpec.scala [hunk 2]
```diff
@@ -59,56 +59,61 @@ class CubeMakerSpec extends TestSuiteBase {
...
       multiplexer,
       checkpointGranularity,
       checkpointInterval,
       checkpointGranularity,
       checkpointTimeAvailability)
-    val dataCube = new CubeOperations(cube, timeDimension, checkpointGranularity)
-
-    testOperation(getEventInput, dataCube.extractDimensionsAggregations, getEventOutput(timestamp), PreserverOrder)
+    val dataCube = new CubeOperations(cube, timeDimensionName, checkpointGranularity)
+    testOperation(getEventInput(sqlTimestamp), dataCube.extractDimensionsAggregations,
```
Comment (danielcsant) [inline aggregator/src/test/scala/com/stratio/sparkta/aggregator/CubeMakerSpec.scala]: It is a small detail and I think It is not neccesary to change for this PR. As we arranged we should set a blank line between test setup and test by itself

Comment (sgomezg) [inline aggregator/src/test/scala/com/stratio/sparkta/aggregator/CubeMakerSpec.scala]: Done :)

**Comment by danielcsant** [comment]
LGTM

**Comment by arinconstrio** [comment]
+1
